src/main/java/com/money_account_service/dtos/response/TransferResponseDto.java

```diff
@@ -0,0 +1,11 @@
+package com.money_account_service.dtos.response;
+
+import lombok.Builder;
+
+import java.math.BigDecimal;
+import java.time.LocalDate;
+
+@Builder
+public record TransferResponseDto(String receiverName, String accountNumber, String title, LocalDate transactionDate,
```

Comment:
I think it's better to retrurn `Instant`, than `LocalDate` which depend on the timezone. I would let the client (frontend) transform the UTC instant to the local date for the user.